azkaban-db/src/main/java/azkaban/db/AzDBOperator.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2017 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ */
+package azkaban.db;
+
+import java.sql.SQLException;
+import org.apache.commons.dbutils.ResultSetHandler;
+
+/**
+ * This interface is to define Base Data Access Object contract for Azkaban. All azkaban
+ * DB related operations must be performed upon this interface. AZ DB operators all leverages
+ * QueryRunner interface.
+ *
+ * @see org.apache.commons.dbutils.QueryRunner
+ */
+public interface AzDBOperator {
+
+  /**
+   * returns the last id from a previous insert statement.
+   * Note: The implementation must catch SQL Exception itself.
+   *
+   * @return the last inserted id in mysql per connection.
+   */
+  Long getLastInsertId();
+
+  /**
+   * Executes the given AZ related SELECT SQL statements.
```

Comment:
Again, I think it would be better to use Azkaban instead of short names. `AZ` or `Az`, etc . The consistency helps.